France-ioi/AlgoreaFrontend
# Feature/observation mode display group permission on item part 2

## Description

Fixes #751

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

...

## Test cases

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/observation-mode-display-group-permission-on-item-part-2/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
  3. And I click on watch group
  4. Then click on "test 2 (Pixal)"
  5. Open «Pixal» can access this activity and click on edit permissions button
  6. In opened dialog change can view permissions and click Proceed 
  7. Open «Pixal» can access this activity and click on edit permissions button
  8. Then I see changed permissions
